cloud/core/src/service.rs

```diff
@@ -0,0 +1,40 @@
+use std::sync::Arc;
+
+use tinc::TincService;
+
+use crate::Global;
+
+pub struct CoreSvc;
+
+impl scuffle_bootstrap::Service<Global> for CoreSvc {
```

Comment:
We should use traits here instead of a known `Global` type. This is because when we want to consolidate all of this into a single binary, it will make it easier.